cmd/convox/start.go

```diff
@@ -84,7 +84,9 @@ func cmdStart(c *cli.Context) error {
 	}
 
 	cache := !c.Bool("no-cache")
-	r := m.Run(dir, app, cache)
+	sync := c.Bool("sync")
```

Comment:
Let's line this up with the `no-cache`. This should be invoked with `convox start --no-sync`